app/_src/gateway/install/docker/index.md

```diff
@@ -439,6 +439,63 @@ docker container rm kong-dbless
 docker network rm kong-net
 ```
 
+## Running Kong in read-only mode
+
+To run Kong in read-only mode, mount a Docker volume to the locations where Kong needs to write data. The default configuration requires write access to `/tmp` and to the prefix path, as provided by the following example:
```

Comment:
This needs a mention of "starting in 3.2.0". We offer no guarantees for previous versions.

Reply:
done. @Kong/docs-maintainers should this be handled with some special version tagging?